### PR TITLE
sql: add docs for stats dynamic update

### PR DESCRIPTION
**Pull request by lamxTyler**
PTAL @coocood @shenli

### CODE REVIEW

File: sql/statistics.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ ANALYZE TABLE TableName INDEX [IndexNameList]
 在发生增加，删除以及修改语句时，TiDB 会自动更新表的总行数以及修改的行数。这些信息会定期持久化下来，
 更新的周期是 5 * `stats-lease`, `stats-lease` 的默认值是 3s，如果将其指定为 0，那么将不会自动更新。
 
+在查询语句执行时，TiDB 会以 `feedback-probability` 的概率收集反馈信息，并将其用于更新直方图和 Count-Min Sketch。
```
Comment (shenli) [inline sql/statistics.md]: How to set/get the `feedback-probability` value?

**Comment by lilin90** [comment]
@shenli PTAL

**Comment by coocood** [comment]
@lamxTyler 
We need to add documents about `auto-analyze-ratio`

File: sql/statistics.md [hunk 2]
```diff
@@ -28,6 +28,10 @@ ANALYZE TABLE TableName INDEX [IndexNameList]
 在发生增加，删除以及修改语句时，TiDB 会自动更新表的总行数以及修改的行数。这些信息会定期持久化下来，
 更新的周期是 5 * `stats-lease`, `stats-lease` 的默认值是 3s，如果将其指定为 0，那么将不会自动更新。
 
+当修改的行数与总行数的比值大于 `auto-analyze-ratio` 时，TiDB 会自动发起 `Analyze` 语句。 `auto-analyze-ratio` 可通过配置文件修改，其默认值是 0，即不开启此功能。
```
Comment (lilin90) [inline sql/statistics.md]: Please delete the extra whitespace after "语句". Thanks!
Usually, **do not** add a whitespace between **Chinese punctuation** and **English/special characters**.

**Comment by lamxTyler** [comment]
PTAL @lilin90

**Pull request merged**